### PR TITLE
FRONT-27: NextJS-pages-directory

### PR DESCRIPTION
**Pull request by ArnoClaude**
## Description of Issue
Moved away from the beta `/app` directory approach to the tried and tested `/pages` directory approach.

## How I implemented
Ran the NextJS wizard by typing `npx create-next-app@latest` in the terminal.

## Other (any extra notes, additional features, problems?, bugs?, performance issues?)
Please squash the commits before merging :) Thanks!

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { Inter } from 'next/font/google'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  return (
+    <main
+      className={`flex min-h-screen flex-col items-center justify-between p-24 ${inter.className}`}
```
Comment (pczern) [inline src/pages/index.tsx:9]: this is an inherently bad design pattern, no separation of concerns as intially intended by HTML5

Comment (pczern) [reply src/pages/index.tsx:9]: but I think PR is ready for merge

**Review by pczern — APPROVED** [review]
Very good approach

**Review by DamianGrosz2 — APPROVED** [review]
Looks good

**Pull request merged**